### ISSUE TITLE
Use message from exception in custom error messages

### ISSUE THREAD

**Issue by dmikhaylov** [flask-restful/flask-restful#274]
In [docs](http://flask-restful.readthedocs.org/en/latest/extending.html#define-custom-error-messages) I see that I'm able to specify just one message per exception using that dictionary.

But what if I need to pass more informative message? For example, If there are some validation errors, I'd like to know what value is actually wrong.

**Comment by mjmare** [comment]
I agree with @jgehrcke that a more dynamic way of handling errors would be useful. And if it uses a technique similar to core Flask like @api.errorhandler that would be preferable.

**Comment by danijar** [comment]
@jgehrcke, [this implements](https://gist.github.com/danijar/827bee07350a96bbf35b) exactly your syntax. Works well for me. Hope it [gets integrated](https://github.com/flask-restful/flask-restful/pull/544) into Flask RESTful soon.
